### PR TITLE
add missing voice and verify code snippets

### PR DESCRIPTION
**Pull request by AlexLakatos**

### CODE REVIEW

**Review by tbedford — APPROVED** [review]

**Comment by tbedford** [comment]
Alex you made some additions after requesting a review so adding Do not merge for now.

**Comment by AlexLakatos** [comment]
While browsing randomly I realized @cr0wst 's list was missing this one verify request with workflow. I think it's good to merge now.

File: voice/make-call-ncco.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+require('dotenv').config({path: __dirname + '/../.env'})
+
+const TO_NUMBER = process.env.TO_NUMBER
```
Comment (marklewin) [inline voice/make-call-ncco.js:3]: `TO_NUMBER` is not listed in `.env`. `NEXMO_TO_NUMBER` is though. (As is `RECIPIENT_NUMBER` (twice))

**Review by marklewin — CHANGES_REQUESTED** [review]
I think there is some confusion about variable naming here? In the "Contribute" section of the docs we say that we prefer `TO_NUMBER`, but `TO_NUMBER` isn't in `.env` and `NEXMO_TO_NUMBER` is (used in the "split audio" snippet).

File: voice/make-call-ncco.js (hunk 1 above)
Comment (tbedford) [reply voice/make-call-ncco.js:3]: Well spotted Mark, I missed that. [This](https://developer.nexmo.com/voice/voice-api/code-snippets/before-you-begin#numbers) also specifies `TO_NUMBER`.

Comment (AlexLakatos) [reply voice/make-call-ncco.js:3]: I've spotted it in the other voice snippets, but I forgot to get back to this and change it. I've updated it in #68

**Pull request merged**